pre_processing/setup.F90

```diff
@@ -840,8 +849,16 @@ subroutine setup_avhrr(args, channel_ids_user, channel_info, verbose)
       j = index(str1, '/', back=.true.)
       str2 = str1
       str1 = str1(1:j-1)
-      str2 = str2(j+1:)
-      args%platform = trim(adjustl(str2))
+      str2 = adjustl(str2(j+1:))
+      if (str2(1:4) == 'noaa') then
+         args%platform = 'NOAA-' // trim(str2(5:))
+      else if (str2(1:6) == 'metopa' .or. str2(1:7) == 'metop02') then
+         args%platform = 'Metop-A'
+      else if (str2(1:6) == 'metopb' .or. str2(1:7) == 'metop01') then
+         args%platform = 'Metop-B'
```

Comment:
Could you add MetOp-C to this?
```
      else if (str2(1:6) == 'metopc' .or. str2(1:7) == 'metop03') then
         args%platform = 'Metop-C'
```

Reply:
Any idea which of A or B this one looks like?